docs/hubs/request-money.html

```diff
@@ -27,7 +29,7 @@ <h2 id="request-money">
             <div class="title">SmartScan</div>
```

Comment:
How about changing the cursor to a pointer so that it indicates the card is clickable

![Screen Shot 2022-07-11 at 5 09 23 PM](https://user-images.githubusercontent.com/12268372/178271621-df99e5ff-5cfb-4893-9069-0087bcc13fa7.png)

Reply:
Good catch Maria! Updating it